foundations/installations/installing_node.md

```diff
@@ -1,12 +1,17 @@
 ### Introduction
 
-Node is a JavaScript interpreter that doesn't run in the browser, there are many tools that require this to be installed. Testing frameworks and bundlers are two of many.
+Node.js is a JavaScript runtime environment, that means that now you can run JavaScript code on the server side also!
```

Comment:
I’m not sure if this early on we need to explain this to learners- it’s way over a beginner’s head re: “high throughput”

Reply:
The previous definition was wrong! Node is not is runtime environment not an interpreter.
If one has done the foundations and JavaScript part of the project then those words won't be much of a problem.

Reply:
The problem is, these instructions come prior to the Javascript course for most folks. I don’t disagree with updating it to refer to it as a runtime though.

Reply:
I agree with @twalton83 in regards to this particular change.

"Node is a JavaScript runtime environment that allows you to run JavaScript outside of your web browser, which will prove necessary throughout your web development journey" or something similar.

I don't think it's worth introducing anything in this particular spot or at the very least it needs to be framed in a way that doesn't raise 3 more questions.